specs/corla.pvs

```diff
@@ -0,0 +1,923 @@
+%%% model of the Colorado Risk Limiting Audit (CORLA) tool.
```

Comment:
I agree... but then, where does that leave the "easiest approach" @nealmcb mentioned above? (that is, why are we still commenting on a closed PR?)